backend/libs/restrictions/src/restrictions.service.ts

```diff
@@ -29,4 +30,39 @@ export class RestrictionsService {
     async getCurfewTime() {
         return await this.queryBus.execute(new GetCurfewTimeQuery());
     }
+
+    @Cron(CronExpression.EVERY_DAY_AT_MIDNIGHT)
+    async extendCurfews() {
+        // fetch curfew
+        const curfew = await this.getCurfewTime(); // this is where we will get the time when this method is fully implemented
+        console.log('curfew is: ', curfew); // here we are just checking what is in the db 
+
+        const _curfewTime = '2022-08-25T20:21:25.034Z'; // this should be the value we receive above 
+        const _signInDate = '2022-08-25T20:21:25.034Z'; // date person signed in regardless of curfew
+        const curfewDate: any = new Date(_curfewTime);
```

Comment:
What is the curfew date referring to? Don't we just need the signInDate? We only need to know when an user signs in. The curfew from the database is not a date, but a time value.